client/src/calendar.jsx

```diff
@@ -22,6 +24,7 @@ class Calendar extends React.Component {
     this.handleLeftButtonClick = this.handleLeftButtonClick.bind(this);
     this.handleDayClick = this.handleDayClick.bind(this);
     this.handleSelectedDay = this.handleSelectedDay.bind(this);
+    this.checkBookings = this.checkBookings.bind(this);
```

Comment:
Looks great! If you're going to bind a lot more functions I would check out auto-bind(https://www.npmjs.com/package/auto-bind), it will replace all those lines 23-27 with one and then you can add 21 and 22.

Reply:
Whoa that's pretty useful! I don't have any more functions for calendar component but I will definitely use this package if it does get to be too much. Thanks Eli!